src/styles/base.css

```
@@ -0,0 +1,213 @@
+  font-weight: var(--fw-med);
+  color: var(--fg-2);
+  text-transform: uppercase;
+  letter-spacing: 0.06em;
+}
+
+/* Blinking terminal cursor */
+.caret {
+  display: inline-block;
+  width: 0.55ch;
+  height: 1em;
+  background: currentColor;
```

Stylelint (17.7.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)